autoload/widgets/fzf-history

```diff
@@ -0,0 +1,13 @@
+#!/bin/zsh
+
+selector-init
+
+fc -l 1 | selector-select --tac --tiebreak=index --bind=ctrl-r:toggle-sort $FZF_CTRL_R_OPTS --query="$LBUFFER" +m
```

Comment:
Please change `$FZF_CTRL_R_OPTS` to `$FZF_WIDGETS_OPTS[insert-history]` because keybind is not always `ctrl+r`.